stdlib/file/src/main/java/org/ballerinalang/stdlib/file/utils/FileUtils.java

```diff
@@ -71,7 +71,7 @@ public static ErrorValue getBallerinaError(String error, Throwable ex) {
      * @return Ballerina error object.
      */
     public static ErrorValue getBallerinaError(String error, String details) {
-        return BallerinaErrors.createError(error, populateSystemErrorRecord(details));
+        return BallerinaErrors.createDistinctError(error, FILE_PACKAGE_ID, error, populateSystemErrorRecord(details));
```

Comment:
I think we can get rid of `populateSystemErrorRecord()` method as well. We only need to pass a record to createDistinctError method only if the detail record has are fields other than `message` and `cause`. So since we pass the `message` field to the `populateSystemErrorRecord()`, we can simply pass only the message.

```suggestion
        return BallerinaErrors.createDistinctError(error, FILE_PACKAGE_ID, details != null ? details : UNKNOWN_MESSAGE);
```

Reply:
Also, shall we change the param name from `details` to `message`. And the docs as well